lib/Config.cpp

```diff
@@ -8,15 +8,38 @@ void LibConfig::Config::parseResponse(json const &data)
 {
   std::string status;
   try {
-    status = data.at("REQUEST_STATE").get<std::string>();
+    //status = data.at("REQUEST_STATE").get<std::string>();
+    status = "STILL_NO_STATUS_SENT";
   } catch (...) {
     // if the data received do not contain 'REQUEST_STATE', it's an event
     std::string setting = data.at("SETTING_NAME").get<std::string>();
     std::string newValue = data.at("UPDATE").get<std::string>();
     bool isDelete = data.at("DELETE").get<bool>();
     return;
   }
-  std::cout << "request status " << status << std::endl;
+  // std::cout << "request status " << status << std::endl;
+
+  try {
+    lastRequestedValue = data.at("SETTING_VALUE").get<std::string>();
+    return;
+  } catch (...) {
```

Comment:
I'd add `assert`s in these catch clauses

Reply:
I think you didn't understand. This catch doesn't mean there is an error, but just that this is not a "SETTING_VALUE" in the response. And, then, we try other kind of responses after

Reply:
ah ok. Marking as resolved.